File: templates/api/examples/js/hello.js
var today = new Date()
var curHr = today.getHours()


exports.handler=function(event,context,cb){
    console.log(JSON.stringify(event,null,2))
    if (curHr < 12) {
        message="good morning, "
    } else if (curHr < 18) {
        message="good afternoon, "
    } else {
        message="good evening, "
    }
    event.res.message=message+event.res.message
    cb(null,event)
}

File: templates/master/examples/py/hello.py
import datetime


def handler(event, context):
    currentTime = datetime.datetime.now()
    
    if currentTime.hour < 12:
        message='Good morning, '
    elif 12 <= currentTime.hour < 18:
        message='Good afternoon, '
    else:
        message='Good evening,'
    event['res']['message']=message+event['res']['message']
    
    return event

